OurPlanscape/Planscape
# removes hardcoded route strategy and fixes some issues on the scenario page map

To test locally:
set this flags to false:
```
  "new_home": false,
  "new_planning_area": false,
  "treatments": false
  ```
  
  Navigate back and forward between planning area page and scenario page. The map should display, and it should center and fit bounds within the planning area.
